openSUSE/open-build-service
# Use component to delete users/groups

On the project or package users page, we can delete users or groups with different roles.

Use DeleteConfirmationDialogComponent instead of a partial to render the confirmation dialog.

**How to test?**

- Go to project or package page.
- Add users and groups with any combination of roles.
- Click to remove one user. The deletion should work.
- Click to remove one group. The deletion should work.
